css/style.css

```diff
@@ -288,6 +332,20 @@ iframe {
 		text-decoration: underline;
 	}
 
+	section.details div div {
```

Comment:
Great, I took another look and it makes perfect sense! I was so focused on the style.css file that I wasn't keeping track of layout.css. Many thanks!